tests/www/views/test_views_connection.py

```diff
@@ -54,3 +54,46 @@ def test_prefill_form_null_extra():
 
     cmv = ConnectionModelView()
     cmv.prefill_form(form=mock_form, pk=1)
+
+
+def test_duplicate_connection(admin_client):
+    """Test Duplicate multiple connection with suffix"""
+    conn1 = Connection(
+        conn_id='test_duplicate_gcp_connection',
+        conn_type='Google Cloud',
+        description='Google Cloud Connection',
+    )
+    conn2 = Connection(
+        conn_id='test_duplicate_mysql_connection',
+        conn_type='FTP',
+        description='MongoDB2',
+        host='localhost',
+        schema='airflow',
+        port=3306,
+    )
+    conn3 = Connection(
+        conn_id='test_duplicate_postgres_connection_copy1',
+        conn_type='FTP',
+        description='Postgres',
+        host='localhost',
+        schema='airflow',
+        port=3306,
+    )
+    with create_session() as session:
+        session.query(Connection).delete()
+        session.add_all([conn1, conn2, conn3])
+        session.commit()
```

Comment:
This part above should be a fixture since it’s test setup, not the actual test.

Reply:
Yeah, But this is very specific to this individual test case.
if we add a fixture, we should use this in at least 2 places.